examples/tests/test_env.py

```diff
@@ -0,0 +1,56 @@
+import os
+import sys
+
+from avocado import Test
+
+
+class Env(Test):
+
+    def test(self):
+        """
+        Logs information about the environment under which the test is executed
+        """
+        pid = os.getpid()
+        p_dir = '/proc/%d' % pid
+
+        def get_proc_content(rel_path):
+            try:
+                return open(os.path.join(p_dir, rel_path)).read().strip()
+            except:
+                return "<NOT AVAILABLE>"
+
+        self.log.debug('Process ID: %s', pid)
```

Comment:
RFE: maybe you can also show parent process information and explain it's the runner